lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -537,6 +541,10 @@ export class FormFieldModel extends FormWidgetModel {
         return Array.isArray(options) ? options.filter((option) => this.isValidOption(option)) : [];
     }
 
+    private parseOptions(options: any, optionType: FieldOptionType): FormFieldOption[] {
```

Comment:
`options` has to be of type any? can't we specify it?

Reply:
this function parses pure json so we can suppose that at that time it's any but as an output we will receive FormFieldOptions[]